app/src/main/java/com/nextcloud/talk/api/NcApi.java

```diff
@@ -349,4 +350,9 @@
     Observable<ContactsByNumberOverall> searchContactsByPhoneNumber(@Header("Authorization") String authorization,
                                                                     @Url String url,
                                                                     @Body RequestBody search);
+
+    @PUT
+    Observable<Response<GenericOverall>> uploadFile(@Header("Authorization") String authorization,
+                                          @Url String url,
+                                          @Body RequestBody body);
```

Comment:
via cmd + shift + l you can reformat code.
In this case "@url" should be beneath "@header"